src/main/java/Deadline.java

```diff
@@ -10,11 +10,15 @@ public String getTaskType(){
         return "D";
     }
 
-    public String toPrint() {
-        return super.toStringStart() + "  [D][" + getStatusIcon() + "] " + description + " (by: " + by + ")\n"  + super.toStringEnd();
+    public String printOk() {
+        return super.printOkStart() + "  [D][" + getStatusIcon() + "] " + description + " (by: " + by + ")\n"  + super.printOkEnd();
     }
 
     public String list() {
-        return "     " + taskCount + "." + "[" + getTaskType() + "]" + "[" + getStatusIcon() + "] " + description + " (by: " + by + ")";
+        return "     Here are the tasks in your list:\n" + "     " + taskCount + "." + "[" + getTaskType() + "]" + "[" + getStatusIcon() + "] " + description + " (by: " + by + ")";
```

Comment:
Coding standard: limit code line to 120 chars,
improvement: can use line wrapping at appropriate places in your code